lib/pages/settings_page/settings_page.dart

```diff
@@ -22,7 +22,29 @@ class SettingsPage extends StatelessWidget {
       Button(
         child: const Text('Delete connection'),
         onPressed: () async {
-          storage.deleteConnection(connection);
+          showDialog<String>(
+              context: context,
+              builder: (context) => ContentDialog(
+                    title: const Text('Delete file permanently?'),
+                    content: const Text(
+                      'Are you sure you want to delete this connection?',
+                    ),
+                    actions: [
+                      Button(
+                        onPressed: () {
+                          storage.deleteConnection(connection);
+                          Navigator.pop(context, 'User deleted connection');
+                        },
+                        child: const Text('Delete'),
+                      ),
+                      Button(
+                        onPressed: () {
+                          Navigator.pop(context, 'User canceled dialog');
```

Comment:
Second parameter is cancel dialog meta, maybe it was short word or several words with underscore as a key in separate constants file

Reply:
i dont understand what you want change...

Reply:
i just copy example from here https://bdlukaa.github.io/fluent_ui/#/popups/content_dialog

Reply:
well maybe do it with general localizaton?